modules/atomic-widgets/prop-types/base/array-prop-type.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropTypes\Base;
+
+use Elementor\Modules\AtomicWidgets\PropTypes\Concerns;
+use Elementor\Modules\AtomicWidgets\PropTypes\Contracts\Persistable_Prop_Type;
+use Elementor\Modules\AtomicWidgets\PropTypes\Contracts\Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+abstract class Array_Prop_Type implements Persistable_Prop_Type {
+	const TYPE = 'array';
```

Comment:
We should find better name, maybe data type?

Reply:
The problem is that `union` and `plain` is not a real data type, maybe `kind`? or `class`? or `group`? @StyleShit

Reply:
`kind` sounds better